buildinfo.cmake

```diff
@@ -48,10 +48,17 @@ function(create_build_info)
 	else()
 		set(FATDB10 0)
 	endif()
+	if (EVMJIT)
+		set(EVMJIT10 1)
+	else()
+		set(EVMJIT10 0)
+	endif()
```

Comment:
I don't want to touch it. The buildinfo.cmake looks like very much overdesigned.